python-avd/pyavd/_eos_designs/schema/schema_fragments/digital_twin.schema.yml

```
@@ -48,6 +48,12 @@ keys:
               Specifies if the ACT Digital Twin device is deployed with direct access to the Internet.
               This option applies only to the 'cloudeos' and 'veos' node types and will be ignored for all other ACT node types.
               ACT does not provide direct Internet access to cloudeos or veos devices by default.
+      default_interfaces_of_original_platform:
+        type: bool
+        default: false
```

Let's do it by default as it seems like the more useful case.
```suggestion
        default: true
```

---

Or maybe reverse the behavior and key name so the default is false `use_default_interfaces_of_digital_twin_platform: <bool; default=false>`

---

Changed according to the later recommendation

---

Reopening this. I think `use_default_interfaces_of_digital_twin_platform` is a bit misleading. It implies we're choosing a different data model, when the code is actually just overriding the platform lookup key for the _same_ `default_interfaces` model.

I think a more accurate name would be `use_physical_platform_for_default_interfaces`. This also makes more sense with the default set to `true`, which enables the magic. Setting it to `false` would then enforce the strict platform validation, which feels more intuive.

---

Not sure. Term "physical" may as well be confusing as we are not picking here between physical and virtual but instead between original platform (which can be already virtual) and "digital twin" platform

---

Since it is harder to identify what we talk about when we refer to the "real" platform, it is simpler to refer to the "digital_twin platform", since that is pretty clear.